src/ray/object_manager/plasma/plasma_allocator.cc

```diff
@@ -90,6 +110,9 @@ absl::optional<Allocation> PlasmaAllocator::Allocate(size_t bytes) {
 }
 
 absl::optional<Allocation> PlasmaAllocator::FallbackAllocate(size_t bytes) {
+  if (!kFallbackEnabled) {
```

Comment:
Just curious but what does k mean here? lol

Reply:
k for constants https://google.github.io/styleguide/cppguide.html#Constant_Names